src/pages/[id].js

```diff
@@ -0,0 +1,8 @@
+import { useRouter } from 'next/router';
+
+export default function Details() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  return <h1>{id}</h1>;
+}
```

Comment:
This file is new in the PR. Is there a `[id].jsx` that already exists? If so, we can delete this file.